packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -63,7 +63,9 @@
 					top: `${node.y}px`,
 					left: `${node.x}px`
 				}"
+				@dragstart="nodeDragging = true"
```

Comment:
overwriting a const
`Type 'true' is not assignable to type 'false'.ts(2322)
`

Reply:
weird ... I fixed that with my last commit...

Reply:
Its declared as a constant and not with `let` or a `ref` so you are unable to reassign 
Your last commit may not have been pushed? I only see 2 commits from ~10 hours ago

Reply:
Yes ... but okay weirdly enough the pre-commit hook thinks otherwise. I think I managed to trick it this time though.

```
Running secrets encryption check
✔ Preparing lint-staged...
✔ Hiding unstaged changes to partially staged files...
✔ Running tasks for staged files...
✖ Prevented an empty git commit!
↓
  ✖ lint-staged failed due to a git error.
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

  ⚠ lint-staged prevented an empty git commit.
  Use the --allow-empty option to continue, or check your task configuration

husky - pre-commit hook exited with code 1 (error)
```

Reply:
Oh ... I think prettier changed my `let` to a const` 😠